src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
+ * and returns an objectWithStyles where CSS properties are keys
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStrings = sourceString.split(';');
+  const objectWithStyles = {};
+
+  for (const el of arrayOfStrings) {
+    const currentKey = el
+      .slice(0, el.indexOf(':'))
+      .trim()
+      .replace(/_/g, '-');
+    const currentValue = el
```

Comment:
```suggestion
      .replace(/_/g, '-');
      
    const currentValue = el
```
el is a bad variable name